### PR TITLE
Mix campaign improvements

### PR DESCRIPTION
**Pull request by soleti**
This PR includes two main changes:

- `FlateMinus` and `NoPrimaryPBI` stages added to the POMS configuration
- Fixed `IPAStop` fcl files

### CODE REVIEW

File: JobConfig/primary/IPAStopParticle.fcl [hunk 1]
```diff
@@ -0,0 +1,52 @@
+#
```
Comment (brownd1978) [inline JobConfig/primary/IPAStopParticle.fcl:1]: This is almost identical to TargetStopParticle. I suggest to create StopParticle.fcl with the common lines, and #include that in both TargetStopParticle and IPAStopParticle, then add the few lines that are different.

File: MDC2020/gen_Mix.sh [hunk 2]
```diff
@@ -20,11 +20,11 @@ is necessary to provide, in order:
...
 - the campaign version of the input file [campaignInput],
 - the campaign version of the output file [campaignOutput],
-- the number of events per job [nEventsPerJob],
-- the number of jobs [nJobs].
+- the number of events per job [nEventsPerJob] (needed only for NoPrimary),
+- the number of jobs [nJobs] (needed only for NoPrimary).
 
 Example:
-    gen_Mix.sh CeEndpoint MDC2020 k m 1000 100
+    gen_Mix.sh CeEndpoint MDC2020 k m
```
Comment (brownd1978) [inline MDC2020/gen_Mix.sh:28]: thanks for updating the comment

File: JobConfig/primary/IPAStopParticle.fcl (hunk 1 above)
Comment (soleti) [reply JobConfig/primary/IPAStopParticle.fcl:1]: Ok, I created `StopParticle.fcl`, I think it should be fine now

**Review by brownd1978 — APPROVED** [review]

**Comment by brownd1978** [comment]
@FNALbuild run build test

**Review by sophiemiddleton — APPROVED** [review]
Looks good, yes I think the PDGId is for the mother now, I see you fixed it so should be fine

**Pull request merged**